README.md

```diff
@@ -63,11 +63,12 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello Microverse 1] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
 
```

Comment:
[OPTIONAL]
- [ ] Kindly remove any part like this one, this is just for your guidance, leaving it may seem inappropriate and unprofessional.